downstream/modules/platform/con-operator-custom-resources.adoc

```diff
@@ -7,7 +7,7 @@ You can define custom resources for each primary installation workflows.
 //[Jameria] Moved this topic from supported installation section to custom resources since that's what the cross-referenced topic links to in the appendix (Custom resources appendix)
 == Modifying the number of simultaneous rulebook activations during or after {EDAcontroller} installation
 
-* If you plan to install {EDAName} on {OCPShort} and modify the number of simultaneous rulebook activations, add the required `EDA_MAX_RUNNING_ACTIVATIONS` parameter to your custom resources. By default, {EDAcontroller} allows 12 activations per node to run simultaneously. See the example in appendix link:{URLOperatorInstallation}#eda_max_running_activations[EDA_MAX_RUNNING_ACTIVATIONS]. 
+* If you plan to install {EDAName} on {OCPShort} and modify the number of simultaneous rulebook activations, add the required `EDA_MAX_RUNNING_ACTIVATIONS` parameter to your custom resources. By default, {EDAcontroller} allows 12 activations per node to run simultaneously. See the example in appendix link:{URLOperatorInstallation}/appendix-operator-crs_performance-considerations#eda_max_running_activations[EDA_MAX_RUNNING_ACTIVATIONS]. 
```

Comment:
Same, we should ensure this is understood to be a global value for OCP

Reply:
This is already covered in this section actually. I think what gets missed when we only show the update we made -- in this case, the update to the link because it's not going to the proper section -- it limits the entire context of the information. When you expand the other section above and below it, you see the entire picture. For the sake of envisioning the information as it is currently contained in the [Installing on the OpenShift Container Platform](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/installing_on_openshift_container_platform/index) document, here's a screen shot showing the information about the global value for OCP in a note contained under section [1.4.1. Modifying the number of simultaneous rulebook activations during or after Event-Driven Ansible controller installation](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/installing_on_openshift_container_platform/index#modifying_the_number_of_simultaneous_rulebook_activations_during_or_after_event_driven_ansible_controller_installation): 

![Screenshot From 2024-12-06 09-39-35](https://github.com/user-attachments/assets/39035c06-e53d-46ed-a2fc-99c1fdce710c)